docs/datamodel/index.rst

```diff
@@ -1,50 +1,116 @@
 .. eql:section-intro-page:: datamodel
 
-==========
-Data Model
-==========
-
-In this section you'll find an overview of the fundamental types, objects,
-and concepts for EdgeDB, a relational database with strongly typed schema.
-Here's a quick overview of the links below:
-
-Types in EdgeDB include your own **Object Types** (e.g. *User*) and
-**Abstract Types** for other types to extend (e.g. ``HasEmailAddress`` for
-``User`` and others can inherit), plus **Scalar Types** with single values
-(``str``, ``int64``, etc.) and **Collection Types** like **arrays** and
-**tuples** for multiple values.
-
-Start putting your **Object Types** together with **properties** and
-**links**. Then build on them with items like **annotations** (readable
-notes for others), **constraints** to set limits (e.g. maximum length,
-minimum value, or even create your own), **indexes** for faster querying,
-and **computed** properties or links to define useful expressions
-(e.g. ``property email := .user_name ++ '@' ++ .provider_name``).
-
-**Expression Aliases** let you use existing types under new names to build
-on them without touching the original -- both in your schema or on the fly
-inside a query. You can also create your own **functions**, strongly typed
-along with everything else in EdgeDB. All this goes into the schema under
-one or more **modules** (namespaces).
+========
+Concepts
+========
 
+In this section you'll find an overview of EdgeDB, a relational
+database with strongly typed schema, starting from the database server
+all the way to the fundamental types, objects, and concepts for
+EdgeDB.
 
 .. toctree::
     :maxdepth: 3
     :hidden:
 
     overview
-    objects
+    objects/index
     scalars/index
     colltypes
     abstract
     functions
     links
     props
+    linkprops
     computables
     indexes
     constraints
     aliases
     annotations
-    modules
-    databases
     extensions
+
+
+Instances
+=========
+
+A running EdgeDB server is called an **instance**. Typically, you
+would create an instance for a :ref:`project
+<ref_cli_edgedb_project>`. You can :ref:`start
+<ref_cli_edgedb_instance_start>`, :ref:`stop
+<ref_cli_edgedb_instance_stop>`, and otherwise manage your instances
+using :ref:`ref_cli_edgedb_instance` commands.
+
+.. _ref_datamodel_databases:
+
+Databases
+=========
+
+An EdgeDB instance can have multiple databases in it. Every instance
+is created with an empty database called "edgedb". If you wish to add
+more, you can use the :eql:stmt:`CREATE DATABASE` EdgeQL command.
+Conversely, the :eql:stmt:`DROP DATABASE` command removes a database.
+
+The following command will get a list of all databases present in the
+instance:
+
+.. code-block:: edgeql
+
+    SELECT sys::Database.name;
```

Comment:
Perhaps mention `\l` in the interactive shell?

Reply:
Added.